src/Cameras/babylon.camera.ts

```diff
@@ -116,16 +120,16 @@
         public viewport = new Viewport(0, 0, 1.0, 1.0);
 
         /**
-        * Restricts the camera to viewing objects with the same layerMask.
-        * A camera with a layerMask of 1 will render meshes with no layerMask and meshes with a layerMask of 1.
-        */
+         * Restricts the camera to viewing objects with the same layerMask.
+         * A camera with a layerMask of 1 will render meshes with no layerMask and meshes with a layerMask of 1.

```

Comment:
Actually the camera will display all meshes where mesh.layerMask & camera.layerMask!== 0